apps/site/assets/ts/schedule/components/schedule-finder/daily-schedule/ServiceOptGroup.tsx

```diff
@@ -67,7 +68,19 @@ const ServiceOptGroup = ({
         }
 
         if (service.id === todayServiceId) {
-          optionText += " (now)";
+          // if it's the only service this rating, this service might not
+          // technically be now - adjust text for that
+          const now = Date.now();
+          if (isInCurrentService(service, new Date(now))) {
+            optionText += " (now)";
+          } else {
+            optionText += ` (Starting ${startDate.toLocaleDateString("en-US", {
```

Comment:
nit-pick(non-blocking): I think we should use a single library for formatting time to keep things consistent (`date-fns`)